phx/results/results_processor.py

```diff
@@ -55,6 +70,37 @@ def _normalize_row(self, row):
         row = tuple(c.strip() if isinstance(c, str) else c for c in row)
         return tuple(None if c == "" else c for c in row)
 
+    def _format_row(self, row):
+        [
+            first_name,
+            surname,
+            sex,
+            age_category,
+            date,
+            distance,
+            race,
+            location,
+            overall_position,
+            _age_position,
+            gender_postion,
+            _best,
+            _age_grading,
+            _club_record,
+            time,
+        ] = row
+
+        position = gender_postion if gender_postion else overall_position
```

Comment:
Good spot, we've done this inconsistently in the past... 

I think we generally always use the overall position, and then (sometimes) also have an extra column for gender position, if it was available. 

I don't have a strong opinion either way, if you have a preference...

Reply:
Interesting, yeah my instinct was to prefer the gender position because it celebrates the women's achievements a bit more. Maybe I'll the other positions in too though so we've got options